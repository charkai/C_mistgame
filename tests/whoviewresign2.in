place C9
who
view
resign
