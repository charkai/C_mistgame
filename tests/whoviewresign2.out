W
B16,xx.....xx.....xx.....xx.....xx.....xx.....xx.....
Black wins!
C9
Thank you for playing!
